### DISCUSSION TITLE
Decision on approach to handle strand info for RNAseq pipeline?

### DISCUSSION THREAD

**Discussion by kubu4** [RobertsLab/resources#1438] [Q&A]
BACKGROUND:

I'm attempting to run the NF-Core RNAseq pipeline. Part of the pipeline runs RSEM. RSEM works off of a GTF file. GTF files have a column dedicated to feature strands. Generally, a feature is found on the `+` or the `-` strand. However, the GTF file spec indicates that this column could also have a `.` to indicate no strand info for a given feature. However, RSEM is written to only process a `+` or `-` and will throw an error when encountering any other character.

I'm attempting to use the [Panopea-generosa-v1.0.a4.gff3](https://gannet.fish.washington.edu/Atumefaciens/20191105_swoose_pgen_v074_renaming/Panopea-generosa-v1.0.a4.gff3). The issue is that there are a set of annotations for tRNA that have a `.` in the strand column. This causes the pipeline to fail:

```
Scaffold_02	GenSAS_5d82b316cd298-trnascan	tRNA	27467	27541	38.1	.	.	ID=21513.GS22252506.PGEN_.tRNA00000001;Name=Thr;anti_codon=CGT;gene_no=1376;
Scaffold_02	GenSAS_5d82b316cd298-trnascan	tRNA	257303	257377	51.3	.	.	ID=21513.GS22252506.PGEN_.tRNA00000002;Name=Ala;anti_codon=CGC;gene_no=1377;
Scaffold_02	GenSAS_5d82b316cd298-trnascan	tRNA	334526	334615	27.0	.	.	ID=21513.GS22252506.PGEN_.tRNA00000003;Name=Ser;anti_codon=CGA;gene_no=1378;
```

---

QUESTION(S):

So, my question is what do you think I should do to handle this? Should I:

- Drop tRNA annotations?

OR

- Convert tRNA strand column to all the same strand?

Both of these options will allow the pipeline to proceed, but I'm not sure what the overall impacts will be for differential gene expression analysis. I'm guessing there will be no impact, as these aren't genes/mRNAs, so will likely be ignored when it comes to the actual differential gene expression analysis, but wanted to see what other people thought.

Of course, I could run the pipeline a couple with each of the proposed changes above and see how the results compare... Just so I can get the pipeline running, I'll probably do this while waiting for feedback.

**Comment by sr320** [comment]
I am not sure I have much concern of tRNA... but as you suggest I would just do the two ways you have suggested.
